<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteWindow</class>
 <widget class="QMainWindow" name="DeleteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeleteWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Yeni klasör (3)/icons.qrc">
    <normaloff>:/newPrefix/pngwing.com.png</normaloff>:/newPrefix/pngwing.com.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
border-image: url(:/arkaplan/duvar-resimleri-kutuphane-kitap-raf-arka-plan-vektor.jpg.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>550</width>
      <height>450</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Author</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Release</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Page</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Enter THe Book Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deleteName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>Exp. Atik Ali</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finalDeleteButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Remove </string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteMnRt">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Return Main Page</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Yeni klasör (3)/icons.qrc"/>
  <include location="icons/arkaplan.qrc"/>
 </resources>
 <connections/>
</ui>
